src/pretix/static/pretixpresale/js/widget/widget.js

```diff
@@ -775,6 +809,28 @@ Vue.component('pretix-widget-event-form', {
             } else {
                 this.$root.view = "weeks";
             }
+        },
+        calculate_buy_disabled: function() {
```

Comment:
Shouldn't this be a ``method`` instead of a ``computed``?

Reply:
I believe it is :), however the diff folding happens to be unfortunate at this point.